src/game/client/image.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file
+ *
+ * @author Smileynator
+ *
+ * @brief Handles Image coordinates and references for asssistance in display of images.
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+
+#include "always.h"
+#include "image.h"
+
+const char *imageStatusNames[3] = {"ROTATED_90_CLOCKWISE", "RAW_TEXTURE", nullptr};
```

Comment:
was likely a static in the function that uses it, also s_imageStatusNames if you make a global, _imageStatusNames if you put it in the function